setup.py

```diff
@@ -265,7 +265,9 @@ def setup_package():
                     package_data={'': ['*.pxd']},
                     **extra_setuptools_args)
 
-    commands = [arg for arg in sys.argv[1:] if not arg.startswith('-')]
+    # Exclude from the list of arguments options and temporary directories
+    commands = [arg for arg in sys.argv[1:] if not arg.startswith('-') and
+                not os.path.isdir(arg)]
```

Comment:
Or even `os.path.exists`?

Reply:
I am worried about the extra system calls and the facts that we might have other kinds of command arguments that are not file that we should neither treat as commands themselves.